app/libs/twitter_processor.rb

```diff
@@ -0,0 +1,30 @@
+require "twitter"
+
+class TwitterProcessor
+  def initialize(twitter_link:, number_of_tweets:)
+    @twitter_link = twitter_link
+    @count = number_of_tweets
+  end
+
+  def process
+    screen_name = @twitter_link.sub('https://twitter.com/', '')
+    scrape_tweets(screen_name)
+  rescue Twitter::Error
+    nil
+  end
+
+  private
+
+  def initialize_twitter_client
```

Comment:
Any better place for this twitter gem configuration? Some how putting it in initializer didn't work and on the internet i only found examples where it is configured straight in classes or controllers